### PR TITLE
Added NetMHCStab to pipeline

### PR DESCRIPTION
**Pull request by agraubert**
See #138

### CODE REVIEW

File: pvacseq/lib/netmhc_stab.py [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+from time import sleep
+from lib.main import split_file
+
+cycle = ['|', '/', '-', '\\']
+methods = ['cterm', '20s']
+
+def main(args_input = sys.argv[1:]):
+    parser = argparse.ArgumentParser("pvacseq net_chop")
+    parser.add_argument(
+        'input',
```
Comment (susannasiebert) [inline pvacseq/lib/netmhc_stab.py]: I would rename this (and `output`) to `input_file` and `output_file`.

File: pvacseq/lib/netmhc_stab.py [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+        type=argparse.FileType('r'),
+        help="Input filtered file with predicted epitopes"
+    )
+    parser.add_argument(
+        'output',
+        type=argparse.FileType('w'),
+        help="Output FASTA filename for putative neoepitopes"
```
Comment (susannasiebert) [inline pvacseq/lib/netmhc_stab.py]: The output file is a TSV, not a FASTA.

**Comment by susannasiebert** [comment]
+1

**Comment by jasonwalker80** [comment]
+1

**Pull request merged**